tests/jest/application-error-boundary/index.test.js

```diff
@@ -0,0 +1,7 @@
+import ApplicationErrorBoundary from '../../../src/application-error-boundary';
+
+describe('application-error-boundary/index', () => {
```

Comment:
I think these are nice sanity checks of exports, but if you look at the generated jest coverage, I don't think these tests affect it....

Reply:
Well the coverage config was jacked in general. Fixed that, but you're right that the index files aren't tracked for coverage.